lib/forms/CoBraLab-Mouse-Weight-Form/index.ts

```diff
@@ -57,32 +57,8 @@ export default defineInstrument({
 
   },
   validationSchema: z.object({
-    mouseWeight: z.string().transform<string | number>((val, ctx) => {
-      const parsed = parseFloat(val);
-      if (isNaN(parsed)) {
-        ctx.addIssue({
-          code: z.ZodIssueCode.custom,
-          message: 'Not a number'
-        });
-        
-
-        // This is a special symbol you can use to
-        // return early from the transform function.
-        // It has type `never` so it does not affect the
-        // inferred return type.
-        return z.NEVER;
-      }
-      if(parsed < 0 || parsed > 45.0){
-        ctx.addIssue({
-          code: z.ZodIssueCode.custom,
-          message: 'Weight has to be in range for 0 to 45 grams'
-        })
-        return z.NEVER;
-      }
-
-      return parsed
-    }),
-    scaleID: z.string(),
+    mouseWeight: z.number().min(1).max(50),
```

Comment:
This is too small, our fat mouse project reached ~70g.

Move this limit to 100

Reply:
ah yes the high fat diet mice, i will increase it to consider them too